### PR TITLE
fix copying up initial contents of the mount point directory

### PR DESCRIPTION
**Pull request by fahedouch**
copying up initial contents of the mount point directory when using `-v` flag

automatically create not found volumes 

fixing https://github.com/containerd/nerdctl/issues/262

Signed-off-by: fahed dorgaa <fahed.dorgaa@gmail.com>

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Thanks, can we have tests?

File: run_mount_test.go [hunk 1]
```diff
@@ -99,7 +99,35 @@ VOLUME /foo
...
+	defer os.RemoveAll(buildCtx)
+
+	base.Cmd("build", "-t", imageName, buildCtx).AssertOK()
+
+	//AnonymousVolume
+	base.Cmd("run", "--rm", imageName).AssertOutContains("hi")
+	base.Cmd("run", "-v", "/mnt", "--rm", imageName).AssertOutContains("hi")
+
+	//NamedVolume
+	base.Cmd("volume", "create", "copying-initial-content").AssertOK()
```
Comment (AkihiroSuda) [inline run_mount_test.go:126]: The volume should be created automatically to fix https://github.com/containerd/nerdctl/issues/262

Can be another PR, though

Comment (fahedouch) [reply run_mount_test.go]: I did both @AkihiroSuda

File: pkg/mountutil/mountutil.go [hunk 2]
```diff
@@ -67,7 +68,14 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 			// assume src is a volume name
 			vol, err := volStore.Get(src)
 			if err != nil {
-				return nil, err
+				if strings.Contains(err.Error(), errdefs.ErrNotFound.Error()) {
```
Comment (AkihiroSuda) [inline pkg/mountutil/mountutil.go:71]: Can we use `errors.Is`?

Comment (fahedouch) [reply pkg/mountutil/mountutil.go]: @AkihiroSuda done

**Review by AkihiroSuda — APPROVED** [review]
Thanks
